tournament-CHR.inc

```diff
@@ -0,0 +1,29 @@
+ips_tilehdr CHR_RIGHT,$68
```

Comment:
Its identical for now, but its very likely to change. I found the ifdef sequence particular hard to read in twoplayer-CHR-01.chr.ips.s so did not want to clutter it anymore. Maybe a good solution would be to generate those files with a simple script which gets 0x68 = 0x18, 0x76 = 0x04, ...  as an input? Is there any ASM equivalent to a C++ .inl file?

Reply:
Doh, the `.define` was wrong. I've swapped this to `NEXT_ON_TOP = 0` (-D on the command line defaults to `0` if no value is specified).

I don't really disagree, although really there is only one if/else is that file; the rest are only present for NEXT_ON_TOP. Part of the reason I share the file like I do is to more easily keep track of which tiles are shared. I can also be sure to reuse tile spaces as I can across mods to be more likely to be compatible with other IPS files.

The verbosity itself doesn't really seem to be a problem, at least to me, so I'm not sure generating it gets us much further. In practice, it hasn't been too bad for me because I would make a change to the file and then mostly ignore it, using the built .chr for the day-to-day work.

I think things could improve reasonably if patterns in twoplayer.chr were reorganized a bit, but I've not thought about it enough to know exactly how would be best.